lib/src/screens/ProfilePage.dart

```diff
@@ -22,17 +22,15 @@ class _ProfilePageState extends State<ProfilePage> {
   final List<PostData> _postData = [
```

Comment:
I'm not sure, where the variables do. 
if not use pls remove its.

Reply:
Same here, Is the Postdata list has any variables inside?